issues on upgrade from 0.0.10 -> 0.0.12

Hi there, couple of (hopefully small) issues :)

```
 browser = getattr(playwright, browser_name).launch(**launch_options)
E       TypeError: launch() got an unexpected keyword argument 'slowMo'
```

Also

```
page.wait_for_response('**/api/1.0/blah*')
E       AttributeError: 'Page' object has no attribute 'wait_for_response'
```

quick 5s debugging..

```
page.__class__
<class 'playwright.sync_api._generated.Page'>

(Pdb) dir(page)

[..., 'wait_for_event', 'wait_for_function', 'wait_for_load_state', 'wait_for_selector', 'wait_for_timeout', 'workers']
```

Cheers~

--- a/docs/src/api/class-page.md
+++ b/docs/src/api/class-page.md
@@ -2950,15 +2950,29 @@ return finalResponse.ok();
 ```
 
 ```python async
-first_response = await page.wait_for_response("https://example.com/resource")
-final_response = await page.wait_for_response(lambda response: response.url == "https://example.com" and response.status === 200)
-return final_response.ok
+async with page.expect_response("https://example.com/resource") as response_info:
+    await page.click("input")
+response = response_info.value
+return response.ok
+
+# or with a lambda
+async with page.expect_response(lambda response: response.url == "https://example.com" and response.status === 200) as response_info:
+    await page.click("input")
+response = response_info.value
+return response.ok
 ```
 
 ```python sync
-first_response = page.wait_for_response("https://example.com/resource")
-final_response = page.wait_for_response(lambda response: response.url == "https://example.com" and response.status === 200)
-return final_response.ok
+with page.expect_response("https://example.com/resource") as response_info:
+    page.click("input")
+response = response_info.value
+return response.ok
+
+# or with a lambda
+with page.expect_response(lambda response: response.url == "https://example.com" and response.status === 200) as response_info:
+    page.click("input")
+response = response_info.value
+return response.ok
 ```
 
 ### param: Page.waitForResponse.urlOrPredicate

